tor/helpers/flair.py

```diff
@@ -59,6 +59,32 @@ def _get_flair_css(transcription_count: int) -> str:
         return 'grafeas'
 
 
+def check_promotion(transcription_count):
+
+    if transcription_count == 10000:
+        return 'grafeas-jade'
+    elif transcription_count == 5000:
+        return 'grafeas-topaz'
+    elif transcription_count == 2500:
+        return 'grafeas-ruby'
+    elif transcription_count == 1000:
+        return 'grafeas-diamond'
+    elif transcription_count == 500:
+        return 'grafeas-golden'
+    elif transcription_count == 250:
+        return 'grafeas-purple'
+    elif transcription_count == 100:
+        return 'grafeas-teal'
+    elif transcription_count == 50:
+        return 'grafeas-green'
+    elif transcription_count==1:
```

Comment:
There seem to be some spaces missing here, you can run `black tor/helpers/flair.py` to run our automatic formatting tool on this.

Reply:
Sir I didn't get this point

Reply:
It's just a small formatting issue, it should be `elif transcription_count == 1:`. We have an automatic formatting tool, called `black` in our Poetry environment, so if you used Poetry to set up your project you can use the `black tor/helpers/flair.py` command to fix the formatting for this file

Reply:
Oh I see you already fixed it, nice!